### PR TITLE
Update setup guide for ML integration packages

### PR DESCRIPTION
**Pull request by susan-shu-c**
## Issues

Update some set-up guides due to user questions/confusion
https://github.com/elastic/integrations/issues/9178

## Summary

Recent user reports running into challenges during package installation (ProblemChild, DGA). 

Clarifying expected behavior for some packages.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

## Previous PR on these files
* https://github.com/elastic/detection-rules/pull/3302

### CODE REVIEW

File: rules/integrations/problemchild/defense_evasion_ml_rare_process_for_a_host.toml [hunk 1]
```diff
@@ -41,7 +41,7 @@ The LotL Attack Detection integration detects living-off-the-land activity in Wi
...
 - If using an Elastic Beat such as Winlogbeat, add the LotL ingest pipeline to it by adding a simple configuration [setting](https://www.elastic.co/guide/en/elasticsearch/reference/current/ingest.html#pipelines-for-beats) to `winlogbeat.yml`.
-- If adding the LotL ingest pipeline to an existing pipeline, use a [pipeline processor](https://www.elastic.co/guide/en/elasticsearch/reference/current/pipeline-processor.html).
+- If adding the LotL ingest pipeline to an existing pipeline, use a [pipeline processor](https://www.elastic.co/guide/en/elasticsearch/reference/current/pipeline-processor.html). For example, you can check if your winlogbeat or Elastic Defend (the [default index pattern](https://docs.elastic.co/en/integrations/endpoint#logs) being `logs-endpoint*`) already has an ingest pipeline by navigating to `Data > Index Management`, finding the index (sometimes you need to toggle "Include hidden indices), checking the index's settings for a default or final [pipeline](https://www.elastic.co/guide/en/elasticsearch/reference/current/ingest.html#set-default-pipeline).
```
Comment (sodhikirti07) [inline rules/integrations/problemchild/defense_evasion_ml_rare_process_for_a_host.toml:44]:
```suggestion
- If adding the LotL ingest pipeline to an existing pipeline, use a [pipeline processor](https://www.elastic.co/guide/en/elasticsearch/reference/current/pipeline-processor.html). For example, you can check if your winlogbeat or Elastic Defend (the [default index pattern](https://docs.elastic.co/en/integrations/endpoint#logs) being `logs-endpoint*`) already has an ingest pipeline by navigating to `Data > Index Management`, finding the index (sometimes you need to toggle "Include hidden indices"), checking the index's settings for a default or final [pipeline](https://www.elastic.co/guide/en/elasticsearch/reference/current/ingest.html#set-default-pipeline).
```

Comment (susan-shu-c) [reply rules/integrations/problemchild/defense_evasion_ml_rare_process_for_a_host.toml]: Thank you! Edited in bulk

File: rules/integrations/problemchild/defense_evasion_ml_rare_process_for_a_host.toml [hunk 2]
```diff
@@ -68,10 +68,10 @@ Before you can enable this rule, you'll need to enrich Windows process events wi
...
 - Under Anomaly Detection, click Jobs, and then click "Create job". Select the Data View containing your enriched Windows process events. For example, this would be `logs-endpoint.events.*` if you used Elastic Defend to collect events, or `winlogbeat-*` if you used Winlogbeat.
-- If the selected Data View contains events that match the query in [this](https://github.com/elastic/integrations/blob/main/packages/problemchild/kibana/ml_module/problemchild-ml.json) configuration file, you will see a card for "Living off the Land Attack Detection" under "Use preconfigured jobs".
+- If the selected Data View contains events that match the query in [this](https://github.com/elastic/integrations/blob/main/packages/problemchild/kibana/ml_module/problemchild-ml.json) configuration file, you will see a card for "Living off the Land Attack Detection" under "Use preconfigured jobs". Warning: if the ingest pipeline hasn't run for some reason, such as no eligible data in winlogbeat has come in yet, _you won't be able to see this card yet_. If that is the case, try to troubleshoot the ingest pipeline, and if any ProblemChild predictions have been populated yet.
```
Comment (sodhikirti07) [inline rules/integrations/problemchild/defense_evasion_ml_rare_process_for_a_host.toml:74]:
```suggestion
- If the selected Data View contains events that match the query in [this](https://github.com/elastic/integrations/blob/main/packages/problemchild/kibana/ml_module/problemchild-ml.json) configuration file, you will see a card for "Living off the Land Attack Detection" under "Use preconfigured jobs". Warning: If the ingest pipeline hasn't run for some reason, such as no eligible data in winlogbeat has come in yet, _you won't be able to see this card yet_. If that is the case, try to troubleshoot the ingest pipeline, and check whether any ProblemChild predictions have been generated.
```

**Comment by terrancedejesus** [comment]
@susan-shu-c even though we are only updating the security content of these rules, they still need an updated `updated_date` to reflect when these changes happened.

FYI - Everything except for `[metadata]` is used to generate a SHA256 of the rule which we use to track rule versions so if any of these change, we typically update the date :)

Thanks in advance!

File: rules/integrations/problemchild/defense_evasion_ml_rare_process_for_a_host.toml [hunk 3]
```diff
@@ -68,10 +68,10 @@ Before you can enable this rule, you'll need to enrich Windows process events wi
...
 - Under Anomaly Detection, click Jobs, and then click "Create job". Select the Data View containing your enriched Windows process events. For example, this would be `logs-endpoint.events.*` if you used Elastic Defend to collect events, or `winlogbeat-*` if you used Winlogbeat.
-- If the selected Data View contains events that match the query in [this](https://github.com/elastic/integrations/blob/main/packages/problemchild/kibana/ml_module/problemchild-ml.json) configuration file, you will see a card for "Living off the Land Attack Detection" under "Use preconfigured jobs".
+- If the selected Data View contains events that match the query in [this](https://github.com/elastic/integrations/blob/main/packages/problemchild/kibana/ml_module/problemchild-ml.json) configuration file, you will see a card for "Living off the Land Attack Detection" under "Use preconfigured jobs". Warning: If the ingest pipeline hasn't run for some reason, such as no eligible data in winlogbeat has come in yet, _you won't be able to see this card yet_. If that is the case, try to troubleshoot the ingest pipeline, and check whether any ProblemChild predictions have been generated.
```
Comment (approksiu) [inline rules/integrations/problemchild/defense_evasion_ml_rare_process_for_a_host.toml:74]: I's suggest using "try troubleshooting"

**Review by approksiu — APPROVED** [review]
Looks great, I'd suggest to use "try troubleshooting" instead of "try to troubleshoot".

**Review by Mikaayenson — APPROVED** [review]
LGTM

**Pull request merged**